developer_manual/basics/backgroundjobs.rst

```diff
@@ -85,6 +85,18 @@ or similar things, consider flagging it as time insensitive in the constructor.
 This allows the Nextcloud to delay the job until a given nightly time window so the users
 are not that impacted by the heavy load of the background job.
 
+Configuring parallelism
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Don't recall the syntax, but:

.. versionadded 27

Reply:
ah, right. Heh, yeah, the syntax on these things is really wack